cli/azd/cmd/templates.go

```diff
@@ -125,11 +125,22 @@ func (tl *templateListAction) Run(ctx context.Context) (*actions.ActionResult, e
 		return nil, err
 	}
 
+	// get clickable link for a repo path
+	clickableRepoUrl := func(repoPath string) string {
+		url, err := templates.Absolute(repoPath)
+		if err == nil {
+			return output.WithHyperlink(url, repoPath)
+		} else {
+			return repoPath
+		}
```

Comment:
nit. No need for `else`
```suggestion
		if err == nil {
			return output.WithHyperlink(url, repoPath)
		}
	        return repoPath
```